feature-libs/asm/components/customer-list/customer-list.component.ts

```diff
@@ -75,14 +80,17 @@ export class CustomerListComponent implements OnInit, OnDestroy {
 
   listsEmpty = false;
 
+  enableAsmB2bCustomerList = false;
+
   protected teardown: Subscription = new Subscription();
 
   constructor(
     protected launchDialogService: LaunchDialogService,
     protected breakpointService: BreakpointService,
     protected asmConfig: AsmConfig,
     protected translation: TranslationService,
-    protected asmCustomerListFacade: AsmCustomerListFacade
+    protected asmCustomerListFacade: AsmCustomerListFacade,
+    protected featureConfig: FeatureConfigService
```

Comment:
Constructors need to be properly overridden when adding a new param.  The new param also needs to be optional.  Otherwise this is a breaking change

Reply:
Any suggestion if we wanna import featureconfig in ts without breaking change?